ramls/checkout-lock-storage.raml

```diff
@@ -0,0 +1,95 @@
+#%RAML 1.0
+title: Checkout lock Storage
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: Checkout lock API
+    content: <b>Storage for checkout lock</b>
+
+types:
+  checkout-lock: !include checkout-lock.json
+  checkout-locks: !include checkout-locks.json
+  checkout-lock-request: !include checkout-lock-request.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+
+/check-out-lock-storage:
+  displayName: Check out lock storage
+  get:
+    description: "Get Checkout Lock list"
+    queryParameters:
+      userId:
+        description: Filter locks by user ID.
+        type: string
+      offset:
+        description: The number of items to skip in the result set.
+        type: integer
+      limit:
+        description: The maximum number of locks to return.
+        type: integer
+    responses:
+      200:
+        description: "Return Checkout Lock list"
+        body:
+          application/json:
+            type: checkout-locks
+      422:
+        description: "Invalid Parameters"
+        body:
+          text/plain:
+            example: "Invalid Parameters"
+  post:
+    is: [validate]
+    description: "Creating a lock for the given userid"
```

Comment:
The same inconsistency issue I can see with endpoint descriptions. For example:
- Get Checkout Lock list - "DO SOMETHING"
- Creating a lock for the given userid - "DOING SOMETHING"
Again, I think it makes sense to choose one approach and used it everywhere.

Reply:
Understood. Now I followed a same approach everywhere.